airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/quotes.json

```diff
@@ -20,8 +20,11 @@
       "type": ["null", "string"],
       "format": "date-time"
     },
-    "associations": {
-      "type": ["null", "object"]
```

Comment:
If I understand this connector code correctly:
* Declaring `Quotes.associations = ["deals"]` will change the query parameters to the Quotes endpoint to retrieve associated deals in the response. 
* The `_flat_assocations` method is meant to unnest the associated fields, it will set `record["deals"] = <whatever_deal_data>`
* The `_flat_assocations` pops the `associations` field from the records but this field is declared in the schema, so `associations` is always `null` 

@zkid18 Do you confirm I'm right in my reasoning and we can safely remove this field?

Reply:
@alafanechere thanks for the comments. 
I need to investigate a bit deeper before the final confirmation.